### PR TITLE
Add `osu!mania 4K World Cup 2024: Finals Recap` news post

### PR DESCRIPTION
**Pull request by 0x84f**
should be good to go mostly

### CODE REVIEW

File: news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+This marked the end of South Korea's run in MWC 4K, with a gamble on RC5 ending up being their last hurrah, sealing their fate with a final score of 4–7.
+
+This match was a perfect showcase of a match where the outcome was not affected by skill, but rather the strategy behind picks, bans, and player nerves. While it's time to bid South Korea farewell, China has accomplished their highest placing ever, and will get a shot to make the dream run come true in next week's Grand Finals.
+
+## Score highlights
+
+The end of the tournament draws ever so closer. While the Grand Finals are still a few days away, we still have a couple scores worth highlighting. With that said, let's see how players fared in the last stretch of the tournament!
+
+- ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) surprised us all with their **absolute bonkers** score of 996,402 in [daiki kasho - Place In This World](https://osu.ppy.sh/beatmapsets/2236927#mania/4752380) (RC6) by [Valedict](https://osu.ppy.sh/users/2537082) in their [match against the United States](https://www.twitch.tv/videos/2257605526). A commendable performance despite clashing against other rice titans.
```
Comment (Walavouchey) [inline news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md:84]:
```suggestion
- ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) surprised us all with their **absolutely bonkers** score of 996,402 in [daiki kasho - Place In This World](https://osu.ppy.sh/beatmapsets/2236927#mania/4752380) (RC6) by [Valedict](https://osu.ppy.sh/users/2537082) in their [match against the United States](https://www.twitch.tv/videos/2257605526). A commendable performance despite clashing against other rice titans.
```

File: news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- Another player worth highlighting is ::{ flag=KR }:: [Naaaad](https://osu.ppy.sh/users/10344857), who **dominated** all the rice picks they participated in. Just to name one instance, in the aforementioned [match versus China](https://www.twitch.tv/videos/2257606852), they managed to score 995,194 points in [S.S.H. - Holy Orders (Be Just or Be Dead)](https://osu.ppy.sh/beatmapsets/2250132#mania/4785237) (RC4), also made by [Valedict](https://osu.ppy.sh/users/2537082).
+
+We've had a blast highlighting the best of scores up until now! Nerves are sky-high and adrenaline is pumping through the players' veins. The teams last standing are well aware of the challenge ahead of them. To them, what matters most are the scores they're going to pull up next round, so lets all hope for an exciting climax! Who knows, maybe we'll get to see them perform in the new custom Tiebreaker for Grand Finals.
```
Comment (Walavouchey) [inline news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md:87]: mm i think it's better to leave the mention for below

```suggestion
We've had a blast highlighting the best of scores up until now! Nerves are sky-high and adrenaline is pumping through the players' veins. The teams last standing are well aware of the challenge ahead of them. To them, what matters most are the scores they're going to pull up next round, so lets all hope for an exciting climax!
```

Comment (0x84f) [reply news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md:87]: idm

**Review by Walavouchey — APPROVED** [review]

File: news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md (hunk 2 above)
Comment (MChecaH) [reply news/2024/2024-09-25-osumania-4k-world-cup-2024-finals-recap.md]: No strong opinion. I think that paragraph works well with or without it, so choose whatever stylistically works best.

**Pull request merged**